src/components/ebay-skeleton-loader/index.marko

```diff
@@ -0,0 +1,48 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+static var ignoredAttributes = [
+    "a11yText",
+    "class",
+    "size",
+    "type",
+    "multiline",
+    "count",
+    "columns",
+    "isVisible",
+];
+static var validTypes = ["avatar", "button", "text", "form", "image", "tile"];
+static var validSizes = ["large", "small"];
+$ var size = validSizes.includes(input.size) ? input.size : "default";
+$ var type = validTypes.includes(input.type) ? input.type : "button";
+$ var typeClass = type === "button" ? "skeleton--btn" : `skeleton--${type}`;
```

Comment:
I think this is a sign that we either need to change `type` to `btn` or the class name to `skeleton--button`

Reply:
This was tricky, the whole skin docs refers button as btn. I wish we named btn as button. Not sure why we did it. To keep it consistent, I can change type to `btn` but I really like type to be spelled out as button.

Reply:
I believe this was because of marko 3. (That or following bootstrap)
We can change it to button next major if needed. 
Lets keep it like this for now though.

Reply:
Done. renamed this to `skeleton-button`.